app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -7,14 +7,19 @@ import AddressSearch, {
   transformKeys,
   snakeCase,
   LocationQuery,
-  LOCATIONS_URL,
 } from '@18f/identity-address-search';
 import BackButton from './back-button';
 import AnalyticsContext from '../context/analytics';
 import InPersonLocations, { FormattedLocation } from './in-person-locations';
 import { InPersonContext } from '../context';
 import UploadContext from '../context/upload';
 
+export const LOCATIONS_URL = new URL(
+  '/verify/in_person/usps_locations',
+  window.location.href,
+).toString();
+export const ADDRESSES_URL = new URL('/api/addresses', window.location.href).toString();
```

Comment:
Unsure if it's already clear, but the root issue that I discovered in the Node 18 upgrade is that Node's native implementation of `fetch` used by MSW can't infer a hostname, which is different than how it works in a browser. So `URL` constructor is needed to make sure it's always a full URL, compatible in both browser & Node.

**Browser:**

```
> await fetch('/')
Response {type: 'basic', url: 'https://github.com/', redirected: false, status: 200, ok: true, …}
```

**Node.js:**

```
> await fetch('/')
Uncaught TypeError: Failed to parse URL from /
    at Object.fetch (node:internal/deps/undici/undici:11457:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async REPL1:1:33 {
  [cause]: TypeError [ERR_INVALID_URL]: Invalid URL
```

To the previous point, this might also be something where using Rails helpers could at least make it a little easier to use full URLs without having to rely on `URL` constructor.